src/screens/employer/EmpUserScreen.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import {
+    Text,
+    StyleSheet,
+    View,
+    SafeAreaView,
+    FlatList,
+    TouchableHighlight,
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import API from '../../constants/API';
+
+export default class EmpUserScreen extends React.Component {
+    state = {
+        user: [],
+        jobs: []
+    }
+
+    async getAllJobsName(jobIds) {
+        const jobs = [];
+        for(i = 0; i<jobIds.length; i++) {
+            const jobId = jobIds[i].jobId;
+            const apiCallJob = await fetch(`${API.endpoint}/jobs/${jobId}`);
+            const job = await apiCallJob.json();
+            const jobObj = {
+                uniqueKey: i.toString(),
+                jobName: job[0].name
+            };
+            jobs.push(jobObj);
+        }
+        return jobs; 
+    }
+
+    async componentDidMount() {
+        const {navigation} = this.props;
+        const code = navigation.getParam('code');
+        try {
+            const apiCallUser = await fetch(`${API.endpoint}/users/${code}`);
+            const user = await apiCallUser.json();
+            const apiCallUserJobIds = await fetch(`${API.endpoint}/users/${user[0].code}/jobs`);
+            const userJobIds = await apiCallUserJobIds.json();
+            const jobs = await this.getAllJobsName(userJobIds);
```

Comment:
You shouldn't need to call `getAllJobsName` anymore. I updated the API last week to return all fields instead of just the ids. I think you can just do: `const jobs = await apiCallUserJobIds.json();`